build/build.js

```diff
@@ -5,39 +5,45 @@ const hyperquest  = require('hyperzip')(require('hyperdirect'))
     , fs          = require('fs')
     , path        = require('path')
     , cheerio     = require('cheerio')
-
+    , encoding    = 'utf8'
+    , nodeVersion = process.argv[2]
+    , nodeVersionRegexString = '\\d+\\.\\d+\\.\\d+'
```

Comment:
just make this `/^\\d+\\.\\d+\\.\\d+$/` no reason to create a string latter instead of a regexp now